nbqa/config_parser.py

```diff
@@ -113,16 +120,21 @@ def parse_config_from_file(cli_args: CLIArgs, project_root: Path) -> Optional[Co
     Optional[Configs]
         Configuration read from the file(if any)
     """
-    config: Optional[Configs] = None
-
     for config_file, config_handler in _CONFIG_FILE_HANDLERS:
         file_path: Path = project_root / config_file
 
         if file_path.is_file() and config_handler is not None:
             config = config_handler(cli_args.command, file_path)
 
-        # we found the config. skip other files
-        if config is not None:
-            break
+            # we found the config. skip other files
+            if config is not None:
+                break
+    else:
+        config = Configs()
+
+    for section in CONFIG_SECTIONS:
+        default = DEFAULT_CONFIG[section].get(cli_args.command, None)
+        if not getattr(config, section, None):
```

Comment:
`getattr(config, section, None)` will always return None because config instance does not contain `section` as it is.  All section members are prefixed with `_`.